src/Microsoft.AspNet.Mvc.ModelBinding/Metadata/ModelMetadataDisplayDetails.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.AspNet.Mvc.ModelBinding
+{
+    public class ModelMetadataDisplayDetails
```

Comment:
my only suggestion would be to add a second concrete display details provider that sets any defaults for the remaining `null` properties.  but that would have special cases as well (e.g. likely would mean the `Order` is also `null`-able) and would move the low-level reflection out of `ModelMetadata`.  all in all, my question really was a request for confirmation and not a request for change.